src/Gui/SettingsScreen/SettingCallback.java

```diff
@@ -0,0 +1,8 @@
+package Gui.SettingsScreen;
+
+import javafx.scene.paint.Color;
+import javafx.util.Pair;
+
+public interface SettingCallback {
+    void onSettingChange(int speed, Color color, int classBlockLength, Pair<Integer, Integer> fastBreak, Pair<Integer, Integer> lunchBreak);
```

Comment:
Zouden al deze losse parameters niet beter gecombineerd kunnen worden tot één klasse?

Iets zoals:
```suggestion
    void onSettingChange(ScheduleSettings settings);
    
    class ScheduleSettings {
    	int speed;
    	Color color;
    	int classBlockLength;
    	Pair<Integer, Integer> fastBreak;
    	Pair<Integer, Integer> lunchBreak;
    }
```

Op deze manier is het ook relatief simpel om later een nieuwe setting toe te voegen, voordat we hier straks een methode met 15 parameters hebben.